test/e2e/webdriver/chrome.js

```diff
@@ -6,7 +6,7 @@ const chrome = require('selenium-webdriver/chrome');
  */
 class ChromeDriver {
   static async build({ responsive, port }) {
-    const args = [`load-extension=dist/chrome`];
+    const args = ['load-extension=dist/chrome', '--no-sandbox'];
```

Comment:
I found I had to do this to get CI to pass, as I was getting the following error: "DevToolsActivePort file doesn't exist". Here is a link to a failing build: https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/12078/workflows/e69a57ce-0950-4da6-9a5b-aee777a5f4cc/jobs/327566. I googled for "DevToolsActivePort file doesn't exist" and found [this SO answer](https://stackoverflow.com/questions/50642308/webdriverexception-unknown-error-devtoolsactiveport-file-doesnt-exist-while-t) which seemed to indicate that `--no-sandbox` makes the error go away. That said, I am not sure whether this is the best fix or whether there is a better one?

Reply:
We should have already been setting `--no-sandbox` :thinking: Were we setting it incorrectly I wonder? Here:
https://github.com/MetaMask/metamask-extension/blob/develop/.circleci/scripts/chrome-install.sh#L27

Reply:
But if it works, it's definitely better than what we had before! This error is what stopped us from updating Chrome for so long. We've long wanted to update.